atom/browser/native_window_mac.mm

```diff
@@ -1078,7 +1079,13 @@ static bool FromV8(v8::Isolate* isolate, v8::Handle<v8::Value> val,
       windowLevel = NSDockWindowLevel;
     }
   }
-  [window_ setLevel:windowLevel];
+
+  NSInteger newLevel = windowLevel + relativeLevel;
+  if (newLevel >= 0 && newLevel < CGWindowLevelForKey(kCGMaximumWindowLevelKey)) {
```

Comment:
Not a big fan of silent fallbacks. Ideally it should throw when I'm giving it invalid values, like above `CGWindowLevelForKey(kCGMaximumWindowLevelKey)`, so I would know I'm doing something wrong.